follow/models.py

```diff
@@ -168,3 +168,32 @@ def cancel(self):
         request_cancel.send(sender=self)
         self.delete()
         return True
+
+
+class RemoteFollow(models.Model):
```

Comment:
```suggestion
class RemoteFollower(models.Model):
```
Can we rename this to RemoteFollower before we get confused ourselves in the next PR